plugins/woocommerce-admin/client/homescreen/header-banner-slot/utils.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { Slot, Fill } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { createOrderedChildren, sortFillsByOrder } from '../../utils';
+
+export const WC_HOMESCREEN_HEADER_BANNER_SLOT_NAME =
+	'woocommerce_homescreen_header_banner_item';
```

Comment:
Just learned that we should add `Experimental` somewhere. 😄 

https://github.com/woocommerce/woocommerce/pull/36487#discussion_r1073234410

Reply:
fixed!

Reply:
@rjchow I think the name needs to have `experimental` in it. Something like

```suggestion
	'experimental_woocommerce_homescreen_header_banner_item';
```

Reply:
@adrianduffell I didn't want to put it in the slot name text string because there'll be a breaking change when we remove the 'experimental_'

Reply:
@rjchow I believe that sort of breaking change is ok because it wouldn't have previously been available for 3PD usage yet. However, we would have to carefully co-ordinate the internal usage to handle the change once it graduates out of experimental.